TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/model/AttractionDetail.java

```diff
@@ -0,0 +1,31 @@
+package com.example.travelpointsbackend.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+@Table(name = "attractionDetails")
+public class AttractionDetail implements Serializable {
+    @Serial
+    private static final long serialVersionUID = 3L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long detailId;
+    private Long attractionId;
+    private String descriptionText;
+    private byte[] descriptionAudio; // or Blob
+    private Float entryPrice;
+    private String offers;
```

Comment:
M: use `@Column(<<column_name>>)` for every field (See here: https://www.geeksforgeeks.org/spring-data-jpa-column-annotation/?ref=lbp)